EIPS/eip-634.md

```diff
@@ -3,6 +3,7 @@ eip: 634
 title: Storage of text records in ENS 
 author: Richard Moore <me@ricmoo.com>
 type: Standards Track
+discussions-to: https://github.com/ethereum/EIPs/pull/634
```

Comment:
Thanks for doing! Unfortunately EIP-1 states it can't be a PR. You could just create an issue on this repo though to accept all discussions to.

Reply:
Thanks for your patience.

I've opened an issue on my repo and made these changes and also done the same for EIP-2098 too.